content/home-2020-oct/card_10/en-US.md

```diff
@@ -0,0 +1,11 @@
+---
+# card 10
+title: "Close the YouTube rabbit hole"
+tag_label: "Extensions"
+desc: "Ever regret watching the next recommended video? This extension doesn’t give you a redo, but it might just keep it from happening to others."
+link_url: "https://foundation.mozilla.org/en/campaigns/regrets-reporter/"
```

Comment:
We usually add UTMs when linking to other mozilla.org domains. `?utm_source=www.mozilla.org&utm_medium=referral&utm_campaign=homepage&utm_content=card`